http-server/pom.xml

```diff
@@ -192,6 +192,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.assertj</groupId>
+            <artifactId>assertj-core</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
please move this test dep below `<artifactId>logback-core</artifactId>` (which is a non-test dep)